includes/Modules/Analytics_4.php

```diff
@@ -506,6 +506,20 @@ private function handle_provisioning_callback( $account_id, $account_ticket ) {
 			)
 		);
 
+		if ( Feature_Flags::enabled( 'enhancedMeasurement' ) && $account_ticket->get_enhanced_measurement_stream_enabled() ) {
+			$this->set_data(
+				'enhanced-measurement-settings',
```

Comment:
This is important to use rather than the service directly has we had run into previously here with the google tag syncing. This is because the data method handles errors and the service infra is a bit lower level and does not.